lib/config/folderconfiguration.go

```diff
@@ -111,6 +116,10 @@ func (f FolderConfiguration) Versioner() versioner.Versioner {
 	return versionerFactory(f.ID, f.Filesystem(), f.Versioning.Params)
 }
 
+func (f FolderConfiguration) ModTimeWindowS() int {
```

Comment:
I think this function should be `ModTimeWindow() time.Duration` (and hence also the call sequence all the way down to `ModTimeEqual`) instead of `...S int`. 

The actual config values are typically seconds to avoid billions in the xml file, but given this is a function anyway...